### PR TITLE
PCSX2 GUI: Update of Scandinavian Translations

### PR DESCRIPTION
**Pull request by pgert**

### CODE REVIEW

File: locales/da_DK/pcsx2_Iconized.po [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 # This file is distributed under the same license as the pcsx2 package.
 # Danish translation for pcsx2 (Iconized).
 # Jakob5566 <jakob5566@gmail.com>, 2017.
-# pgert <http://forums.pcsx2.net/User-pgert>, 2019.
+# pgert <http://forums.pcsx2.net/User-pgert>, 2020.
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 1.5.0 - 190526\n"
+"Project-Id-Version: PCSX2 1.5.0 - 200115\n"
```
Comment (lightningterror) [inline locales/da_DK/pcsx2_Iconized.po]: 1.6.0 version instead.

File: locales/da_DK/pcsx2_Main.po [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 # This file is distributed under the same license as the pcsx2 package.
 # Danish translation for pcsx2 (Main).
 # Jakob5566 <jakob5566@gmail.com>, 2017.
-# pgert <http://forums.pcsx2.net/User-pgert>, 2019.
+# pgert <http://forums.pcsx2.net/User-pgert>, 2020.
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 1.5.0 - 190526\n"
+"Project-Id-Version: PCSX2 1.5.0 - 200115\n"
```
Comment (lightningterror) [inline locales/da_DK/pcsx2_Main.po]: Ditto.

File: locales/nb_NO/pcsx2_Iconized.po [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 # This file is distributed under the same license as the PCSX2 package.
 # Norwegian translation for pcsx2 (Iconized).
 # Imre Kristoffer Eilertsen <imreeil42@gmail.com>, 2017.
-# pgert <http://forums.pcsx2.net/User-pgert>, 2019.
+# pgert <http://forums.pcsx2.net/User-pgert>, 2020.
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 1.5.0 - 190526\n"
+"Project-Id-Version: PCSX2 1.5.0 - 200115\n"
```
Comment (lightningterror) [inline locales/nb_NO/pcsx2_Iconized.po]: Ditto.

File: locales/nb_NO/pcsx2_Main.po [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 # This file is distributed under the same license as the PCSX2 package.
 # Norwegian translation for pcsx2 (Main).
 # Imre Kristoffer Eilertsen <imreeil42@gmail.com>, 2017.
-# pgert <http://forums.pcsx2.net/User-pgert>, 2019.
+# pgert <http://forums.pcsx2.net/User-pgert>, 2020.
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 1.5.0 - 190526\n"
+"Project-Id-Version: PCSX2 1.5.0 - 200115\n"
```
Comment (lightningterror) [inline locales/nb_NO/pcsx2_Main.po]: Ditto.

File: locales/sv_SE/pcsx2_Iconized.po [hunk 5]
```diff
@@ -1,13 +1,13 @@
-# Copyright (C) 2019 PCSX2 Team
+# Copyright (C) 2020 PCSX2 Team
 # This file is distributed under the same license as the PCSX2 package.
 # Swedish translation for pcsx2 (Iconized).
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 - 190526\n"
+"Project-Id-Version: PCSX2 - 200115\n"
```
Comment (lightningterror) [inline locales/sv_SE/pcsx2_Iconized.po]: Ditto.

File: locales/sv_SE/pcsx2_Main.po [hunk 6]
```diff
@@ -1,13 +1,13 @@
-# Copyright (C) 2019 PCSX2 Team
+# Copyright (C) 2020 PCSX2 Team
 # This file is distributed under the same license as the PCSX2 package.
 # Swedish translation for pcsx2 (Main).
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 - 190526\n"
+"Project-Id-Version: PCSX2 - 200115\n"
```
Comment (lightningterror) [inline locales/sv_SE/pcsx2_Main.po]: Here as well.

**Comment by turtleli** [comment]
Sorry about this, but we've only just noticed that we were missing a translatable string.

Can you rebase to latest master and translate the additional string as well?

**Comment by pgert** [comment]
Rebase?

**Comment by lightningterror** [comment]
You can update the existing one with just doing a proper rebase, see
https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase
https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request
these will help to know how a rebase is done. Let us know if you need help.

**Comment by pgert** [comment]
Not this again. I'll make a new PR instead (please?).

**Comment by MrCK1** [comment]
If you're unable to rebase leave it for us. Thanks.

**Comment by turtleli** [comment]
If you enable the "Allow edits from maintainer" option a maintainer can handle the rebase for you.

**Comment by pgert** [comment]
Enabled.

**Comment by turtleli** [comment]
Rebased.

**Comment by pgert** [comment]
Can someone merge this now, please?

**Comment by MrCK1** [comment]
We will attend to it when we have time, thanks.

**Comment by lightningterror** [comment]
@pgert can you join us on discord and work with the translator team to better improve the translations, we got feedback that current translations may not actually be good.

**Comment by pgert** [comment]
@MrCK1:
OK, thanx.

@lightningterror:
Discussions regarding the translations can be done in the pcsx2 forum, or through PR's.
I'm not up for bickering on discord.
There's always disagreements, and those who want a more "modern" (anglified) language.

**Comment by refractionpcsx2** [comment]
@pgert can you actually speak all this languages fluently? because we have been getting reports from native speakers that your translations are terrible and we are suspicious that you are just using online translators.

If you aren't fluent in any of these languages PLEASE do not attempt to translate them leave them for other people who are native speakers, we would rather have accurate translations than guess work.

Thanks.

**Comment by RokkumanX** [comment]
**IMPORTANT**

As a guy born in Sweden and is a native Swedish speaker, all I can say is DON'T MERGE! 
I have read through the Swedish translation and most of it is a complete JOKE and it definitely smells Google Translate.

It's full of grammatical errors and some sentences look really foolish and dumb, it really needs to be translated by a professional and not by Google Translate.

**Comment by TheNekOz** [comment]
@RokkumanX
Doesn't even have to be a professional. Anyone with decent language skills, ability to read a dictionary and patience can do translations.

The Norwegian translation is bad enough as it is and most of the changes don't really help with that. I would have completed it and done a pull request a long time ago if I didn't have to prioritize my studies right now. Only have like... Half of Iconised left to translate.

**Comment by pgert** [comment]
@RokkumanX & @TheNekOz:
Is there something about THIS particular PR that stands out?

I have (needless to say) been using wordbooks alot for danish & norwegian.

@TheNekOz:
If you want to take over the norwegian translation, then that would be preferable.

**Comment by TheNekOz** [comment]
> @RokkumanX & @TheNekOz:
> Is there something about THIS particular PR that stands out?
> 
> I have (needless to say) been using wordbooks alot for danish & norwegian.
> 
> @TheNekOz:
> If you want to take over the norwegian translation, then that would be preferable.
Sure, I can take over the Norwegian translation.

As for some errors in this pull request.
You changed Syklusfrekves, to cyklusfrekvens. Good correction with the capitalization, but it's syklus.
Website is Nettside, not Websted. As for GitHub Repository, I don't think Magasin is the.. right word for it. There are no direct translation and multiple variations have to be considered. Some ideas are oppbevaringssted, arkiv, and repositorium (being the most direct translation). I will most likely be going for repositorium as it's a single word and shares pretty much the same meaning as repository.

Memory Card is always translated as MinneKort. One example being "Konverter dette MinneKort til en mappe med individuelle lagrefiler.". I would instead translate it as "Konverter dette minnekortet til en mappe med individuelle lagringsfiler.".

Except for that, there are tons of minor and major translation errors littered across the existing translation. A few Swedish and Danish words here and there, like the word Eksekverbar, meaning Executable. While it seems it's commonly used in Danish, it's honestly my first time even hearing about that word and it ain't even in the Norwegian dictionary. Utførbar is the commonly used word for Executable.

Some sentences are straight up broken, like "At muliggjøre bredbildetriks kan måhende volde problemer" and "Bruk NTFS Komprimering for skapende av dette kort".

I spent way too much time on this comment... Oh well.

**Comment by pgert** [comment]
@TheNekOz:

I changed "Syklusfrekves" into "cyklusfrekvens" because it's required.
The translated strings won't be accepted otherwise.
This is due to an error in POedit (it seems).
It's the "%" sign that causes this problem.
Try it out for yourself.

As for the rest, well I hope you can make it better.

**Comment by RokkumanX** [comment]
I have no formal experience in translating software, emulation software even.

Translating takes passion and time and right now I got neither but as a Swedish guy it's my duty to at least proof read what I will use and from what I read in the translation it sounded robotic and inhuman.

I'm sorry if this is sounding harsh, but it's better to be honest which everyone benefits from in the long run.

**Pull request closed without merging**